src/utils/ErrorHelper.ts

```diff
@@ -118,6 +118,11 @@ Please fill this information with details if relevant.
 ${metadata.error.stack}
 \`\`\`
 
+### React Component Stack
```

Comment:
Can we make this whole section optional based on errorInfo presence? (Is that a useful thing to do? Is it present most of the time, or only sometimes?)

Reply:
AFAIK, it's presence in most cases. When the recovery mode UI is activated by `ErrorBoundary`, `errorInfo` will be set. The only exception that I know is when we activate from `LoadingScreen` due to loading error.

Nonetheless, I can make your suggested change if you want, even when it's presense in most cases. If that's the case, just let me know.

Reply:
I suppose we don't really have much more than our intuition on how many RecoveryScreen views are from `LoadingScreen` vs the rest of the game. I'm guessing it actually tilts towards `LoadingScreen`, but it is also just a guess.